src/components/CompareResults/SubtestsResults/SubtestsRevisionRow.tsx

```diff
@@ -0,0 +1,297 @@
+import { useState } from 'react';
+
+import ExpandLessIcon from '@mui/icons-material/ExpandLess';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import TimelineIcon from '@mui/icons-material/Timeline';
+import { IconButton } from '@mui/material';
+import { style } from 'typestyle';
+
+import { useAppSelector } from '../../../hooks/app';
+import { Strings } from '../../../resources/Strings';
+import { Colors, Spacing, ExpandableRowStyles } from '../../../styles';
+import type { CompareResultsItem } from '../../../types/state';
+import RevisionRowExpandable from '.././RevisionRowExpandable';
+
+const revisionsRow = {
+  borderRadius: '4px 0px 0px 4px',
+  display: 'grid',
+  margin: `${Spacing.Small}px 0px`,
+  // Should be kept in sync with the gridTemplateColumns from TableHeader
+  gridTemplateColumns: '2fr 1fr 0.2fr 1fr 1fr 1fr 1fr 1fr 2fr 0.2fr',
+};
+
+const typography = {
```

Comment:
I added this suggestion in the clean-up [subtask](https://mozilla-hub.atlassian.net/browse/PCF-488) and it will be addressed later on for RevisionRow.tsx